scripts/prepare_compliance_data.py

```diff
@@ -154,12 +172,15 @@ def main():
         help="The directory to output the server-ready data bundle to.")
     parser.add_argument(
         "--inputDirectory", "-i",
-        help="Path to local directory containing compliance dataset.",
-        default='.')
+        help="Path to local directory containing compliance dataset. "
+        "If no directory is provided this script will attempt to "
+        "download the compliance test-data from github",
+        default=None)
     parser.add_argument('--verbose', '-v', action='count', default=0)
     args = parser.parse_args()
     cdm = ComplianceDataMunger(args)
     cdm.run()
```

Comment:
Need a try, finally here or we'll spray temp dirs over the file system when errors occur:
```python
try:
    cdm = ComplianceDataMunger(args)
    cdm.run()
finally:
    cdm.cleanup()
```